<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TanksGameClass</class>
 <widget class="QMainWindow" name="TanksGameClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>620</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>TanksGame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {background-color : black;
color: white;
font-size: 30px;}
QPushButton{ border : 2px solid white; border-radius : 10px; }
QLabel{ border : 2px solid white; }
QPushButton:hover{ color: red; }
QPushButton:pressed{  border-color: red; }
QMenuBar {font-size: 12pt;}
QMenuBar:item:hover {border : 1px solid white;}
QMenu:item {font-size: 12pt; max-width: 12px;}
QMenu:item:hover { border : 1px solid white; }</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>210</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Tanks 1984</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pveButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>210</width>
      <height>75</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Single Player</string>
    </property>
   </widget>
   <widget class="QPushButton" name="conButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>360</y>
      <width>210</width>
      <height>75</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Constructor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pvpButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>260</y>
      <width>210</width>
      <height>75</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Two players</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>460</y>
      <width>210</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>Exit</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout_2">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
   <addaction name="menuAbout_2"/>
   <addaction name="menuExit"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout_2">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TanksGame.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>TanksGameClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>TanksGameClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>516</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
